tests/conftest.py

```diff
@@ -2,17 +2,26 @@
 from virtwho.runner import VirtwhoRunner
 from virtwho.configure import VirtwhoHypervisorConfig
 from virtwho.configure import VirtwhoGlobalConfig
-from virtwho.configure import get_hypervisor_handler
+from virtwho.configure import get_hypervisor_handler, virtwho_ssh_connect
 from virtwho.configure import get_register_handler
 from virtwho.register import SubscriptionManager
-from virtwho import logger
 import pytest
 
 
 mode = config.job.hypervisor
 register_type = config.job.register
 
 
+@pytest.fixture(name='rhel_compose', scope='session')
+def virtwho_test_rhel_compose():
+    return config.job.rhel_compose
+
```

Comment:
Consider global params

Reply:
A really good suggestion and I updated the code, please review, thanks.